### PR TITLE
Response

### PR DESCRIPTION
**Pull request by andreasantarl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+Number  5
+String  'This is a string'
+Boolean  true
+null     let value = null;
+undefined    let name;
```
Comment (berziiii) [inline README.md:24]: `undefined = undefined` When you do `let name` you are declaring the variable, just not assigning it yet. But yes, correct.

File: README.md [hunk 2]
```diff
@@ -28,7 +33,8 @@ c = c + d;
 ```
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
-
+c = 7
+The operators are addition operator, which adds values together, and the assignment operator, which attributes the value of something to a variable.
```
Comment (berziiii) [inline README.md:37]: Correct.

File: README.md [hunk 3]
```diff
@@ -41,7 +47,7 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- solution below -->
+Initially, the value of x is 4 and the value of y is 3. The values then change but not the variables themselves.  X is 3 and the value of y is 10 at the end of the code block.
```
Comment (berziiii) [inline README.md:50]: Correct.

File: README.md [hunk 4]
```diff
@@ -54,6 +60,10 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+The values of these expressions are:
+let weather;   undefined  (there is no value for the variable)
+weather = "sunny";   'sunny'  (the assignment operator gives weather the value of 'sunny')
+weather === "sunny";  true  (weather already has the value of 'sunny', and we're using the comparison operator to see if the vaue is 'sunny', which is true)
```
Comment (berziiii) [inline README.md:66]: Correct.

File: bin/diagnostic.js [hunk 5]
```diff
@@ -1 +1,14 @@
 'use strict';
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
```
Comment (berziiii) [inline bin/diagnostic.js]: let answer = ' ' ;

File: bin/diagnostic.js [hunk 6]
```diff
@@ -1 +1,14 @@
...
+
+while (answer !== '' && (answer < 1 && answer > 10)) {
+  //answer = ask ('Give me a number between 1 and 10' );
+  answer = console.log("You need to guess a number between 1 and 10, nerd ");
+  if (answer >= 1 && answer <= 10) {
+   console.log("You guessed it!");
+   break;
+  }
+}
```
Comment (berziiii) [inline bin/diagnostic.js]: See below:

```javascript
var answer = 'not empty';

while (answer !== '') {
  answer = ask("Guess my secret? ");
  if (answer >=1 && answer <=10) {
   console.log("You guessed it!");
   break;
 }else{
   console.log('Please try a number between 1 and 10')
 }
```

Comment (RDegnen) [inline bin/diagnostic.js:14]:
```
'use strict';
//We'll learn about require later in the course
const ask = require('../lib/ask.js');

var answer = 'not empty';

while (answer !== '') {
  answer = ask("Guess my secret? ");
  if (answer >=1 && answer <=10) {
   console.log("You guessed it!");
   break;
 }else{
   console.log('Please try a number between 1 and 10')
 }
}
```

**Comment by RDegnen** [comment]
4 of 5 not bad

**Pull request closed without merging**